ProcessExporter.py

```diff
@@ -329,8 +328,7 @@ def get_process_name(self):
         if proc_number != 0:
             process_string = "%d_%s" % (proc_number, process_string)
 
-        process_string = "Sigma_%s_%s" % (self.model_name,
-                                          process_string)
+        process_string = f"Sigma_{self.model_name}_{process_string}"
```

Comment:
Since many people (e.g. in CMS) still use the python2 version of MG5, I think keeping it compatible with 2.7 would still be nice... this unfortunately means no f-strings. (I think raw strings are fine?)

Reply:
Ah, bummer. Okay I went back and rebased things so that `pre-commit` is now applying `pyupgrade` with just the `--py3-plus` option and not the `--py37-plus` option.

Reply:
FTR, I [agree with you](https://twitter.com/HEPfeickert/status/1405192438290042882) that the situation is ridiculous, but I don't control the whole CMS software infrastructure, or MG5_aMC@NLO, so if the price to pay for having the software run without crashes is to avoid two f-strings in a small script, so be it...

Reply:
> so if the price to pay for having the software run without crashes is to avoid two f-strings in a small script, so be it

Totally, and I fully understand and appreciate that. To be [clear](https://twitter.com/HEPfeickert/status/1407339275574775811), as single tweets are not great for communicating full ideas, my "disappointment" is with CMS's and ATLAS's states as a whole and the "you" and "users" are the experiments and everyone who has to use and fit into their software.

I think you should feel pretty good though that the CMS uses MoMEMta in their toolchains enough that MoMEMta falls into the group that needs to fit into their software. :+1:

Reply:
Thanks for clarifying! What's also sure is that user frustration helps driving updates, so some of it can help ;)